src/Microsoft.Extensions.Logging.Console/Properties/AssemblyInfo.cs

```diff
@@ -3,6 +3,8 @@
 
 using System.Reflection;
 using System.Resources;
+using System.Runtime.CompilerServices;
 
 [assembly: AssemblyMetadata("Serviceable", "True")]
-[assembly: NeutralResourcesLanguage("en-us")]
\ No newline at end of file
+[assembly: NeutralResourcesLanguage("en-us")]
+[assembly: InternalsVisibleTo("Microsoft.Extensions.Logging.Test")]
```

Comment:
Ok, but there is only test project for all these Console, Debug and EventLog projects...are you suggesting to create different test projects respectively?